src/harness.js

```diff
@@ -393,7 +395,30 @@ class Harness {
   get consoleErrors() {
     return this._state.console
       .filter(msg => msg.type() !== 'log')
-      .filter(msg => msg.text() !== 'Failed to load resource: net::ERR_UNKNOWN_URL_SCHEME');
+      .filter(msg => msg.text() !== 'Failed to load resource: net::ERR_UNKNOWN_URL_SCHEME')
+      .filter(msg => msg.type() !== 'warning');
+  }
+
+  /**
+   * A filtered set of output (neither errors nor warnings) inside of state.console. Useful for easy assertions.
+   * @example
+   * expect(harness.consoleOutput).to.be.empty;
+   */
+
+  get consoleOutput() {
+    return this._state.console
+      .filter(msg => msg.type() === 'log')
+      .filter(msg => msg.text() === 'Failed to load resource: net::ERR_UNKNOWN_URL_SCHEME');
+  }
+
+  /**
+   * A filtered set of warnings inside of state.console. Useful for easy assertions.
+   * @example
+   * expect(harness.consoleWarnings).to.be.empty;
+   */
+  get consoleWarnings() {
```

Comment:
i think this interface is not that valuable and can be removed... people can filter `this.consoleOutput` by all kind of criteria with just one line. are warnings more important than any other criteria?

Reply:
Vestiges of my previous change. Cleanup coming up!